### PR TITLE
FESBAL-49 Create entity flow

### PR DESCRIPTION
**Pull request by santigamo**
## Description

Create entity flow feature

## Checks

- [x] Project Builds
- [x] Project passes tests and checks
- [x] Updated documentation accordingly

### CODE REVIEW

File: src/read-models/recipient-read-model.ts [hunk 1]
```diff
@@ -49,4 +49,25 @@ export class RecipientReadModel {
       entity.referralSheetUrl
     )
   }
+
+  @Projects(Recipient, 'identityDocumentNumber')
+  public static projectRecipientUserByDocumentNumber(entity: Recipient): ProjectionResult<RecipientReadModel> {
```
Comment (samueldominguez) [inline src/read-models/recipient-read-model.ts:54]: why do you call this `entity`?

**Review by samueldominguez — APPROVED** [review]
LGTM but left a comment

File: src/read-models/recipient-read-model.ts (hunk 1 above)
Comment (santigamo) [reply src/read-models/recipient-read-model.ts:54]: It's how boost cli names that param by default!

**Review by samueldominguez — APPROVED** [review]
LGTM!

**Pull request merged**